mozilla/bedrock
# Send scroll event to data layer #15098

## One-line summary

Send scroll event to data layer at 25%, 50%, 75% and 90% of the page.

## Significant changes and points to review

- Adds scroll listener
- Adds event to the dataLayer when thresholds are passed
- Removes listener after all thresholds are passed

!!! This alone is not enough to get the events into GA4 - GTM changes must be made too.

## Issue / Bugzilla link

Requested in #15098 but people keep asking me for these numbers on campaigns that have finished running and it's too late to get them. So this will add the event to all pages.

## Testing

- scroll the page
- use the console to view the dataLayer
- see if the events are in the dataLayer
